google-apis-core/lib/google/apis/core/opentelemetry.rb

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'opentelemetry-api'
+
+module Google
+  module Apis
+    module Core
+      class OpenTelemetry < OpenTelemetry::Instrumentation::Base
+        install { true }
+        present { true }
+      end
+    end
+  end
+end
```

Comment:
With the [addition of the dynamic upgradeable tracer](https://github.com/open-telemetry/opentelemetry-ruby/pull/708) do we still want to make use of this?